src/Components/ImagesTable/ImageLink.js

```diff
@@ -11,41 +11,27 @@ const ImageLink = ({
   imageStatus,
   ...props
 }) => {
-  const scalprum = useScalprum();
-  const hasProvisionig = scalprum.initialized && scalprum.config?.provisioning;
```

Comment:
@amirfefer this was kinda important, because in some environment (test and prod ATM) we do not have the provisioning app and thus provisioning manifest present. This check made sure we render the fallback component in those environments.